power-outage-monitor/README.md

```diff
@@ -0,0 +1,114 @@
+* [Notecarrier-A](https://shop.blues.io/collections/notecarrier/products/carr-al) or [Notecarrier-B](https://shop.blues.io/collections/notecarrier/products/carr-b)
+* Cellular [Notecard](https://blues.io/products/notecard/)
+
+
+## Hardware Setup
+
+1. Assemble Notecard, Notecarrier and LiPo battery as described in our [tutorial](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-b/).
+
+2. Keep the Notecard connected to the computer with the in-browser terminal active until you have completed [Notecard Setup](#sotecard-setup).
+
+## Notecard Firmware
+
+Your notecard should be running [firmware version 3.5.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022) or higher version. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub)
+
+## Cloud Setup
+
+- Sign up for a free account on [Notehub.io](https://notehub.io), and create a new project.
+  `{"req":"hub.set", "product":"com.your-company.your-name:your_product", "mode":"continuous"}`
+- Twillio route: TBD
+
+## Notecard Setup
+
+In these steps, you will use the in-browser terminal to configure your notecard to report power outage events to your Notehub project.
+
+### Configure Notehub Project and Connection Mode
+
+First, we will set the serial number (name) of your notecard and the ProductUID. The ProductUID is the one corresponding to the project you careted in [Cloud Setup](#cloud-setup). Replace the `sn` and `product` values with your own and paste the resulting command into the in-browser terminal.
+
+```json
+{
+  "req": "hub.set",
+  "mode": "periodic",
+  "inbound": 1440,
+  "outbound": 60,
+  "product": "com.your-company:your-product-name",
+  "sn": "your-serial-number"
+}
+```
+
+> This app uses `periodic` mode to minimize power use so that solution can remain online for longer when mains power fails. For more details, see [`hub.set` requests](https://dev.blues.io/notecard/notecard-walkthrough/essential-requests/#notehub-configuration).
+
+### Configure Power Outage Alerts
+
+Paste this command into the in-browser terminal
+
+```json
+{ "req": "card.voltage", "usb": true, "alert": true, "sync": true }
+```
+
+> This instructs the Notecard to monitor USB power and send an alert immediately when a power outage occurs and when power is restored. For more details, see the `Enable USB Power Alerting` example in the [Notecard API reference](https://dev.blues.io/reference/notecard-api/card-requests/#card-voltage) and the 
+
+
+## System Test
+
+In this section, you will power the Notecard via the power brick and simulate a power failure.
+
+### Setup
+
+1. Insert the power brick into a suitable electrical outlet and provide power to the Notecarrier via the micro USB cable.
+
+1. Wait for the Notecard to connect to the cloud.  When connected, there will be a new session event in Notehub. (look for recent `_session.qo` events.)
```

Comment:
This should be 2 and below should be 3.

Reply:
Markdown renderers will number the lists correctly. https://github.com/blues/app-accelerators/tree/mat-power-outage-monitor/power-outage-monitor. 

Of course, if devs are reading markdown from the source it's a little more confusing, but really, do people do that? ;-)

Here's another example. https://raw.githubusercontent.com/blues/app-accelerators/main/low-power-digital-signage/README.md.  The reason for doing this is to make later revisions easier (e.g. inserting a new item in the middle of the list without having to renumber everything.)

Reply:
TIL!